pkg/server/config.go

```diff
@@ -63,10 +64,10 @@ type ServerConfigFile struct {
 	Kubelet      KubeletConfig    `json:"kubelet"`
 }
 
+// Kubelet stores kubelet-specific configuration such as capacity and labels.
 type KubeletConfig struct {
-	CPU    resource.Quantity `json:"cpu"`
-	Memory resource.Quantity `json:"memory"`
-	Pods   resource.Quantity `json:"pods"`
+	Capacity v1.ResourceList   `json:"capacity"`
```

Comment:
We typically can't delete/move fields in this structure since since existing deployments will break.  I don't know what our policy should be here since there might not be any existing deployments out there.

Reply:
> We typically can't delete/move fields in this structure since since existing deployments will break. I don't know what our policy should be here since there might not be any existing deployments out there.

I tested this, and fields that are not in the struct are ignored during decoding. That being said, I can add them back, and if they are used, set the corresponding field(s) in kubelet.capacity.

Reply:
I pushed a commit doing this.